packages/app/src/runner/SpecRunnerDropdown.vue

```diff
@@ -1,23 +1,36 @@
 <template>
   <Popover
+    :key="`${popoverKey}`"
```

Comment:
My Vue knowledge is still very very fresh so I could definitely be off 😸  is the template string needed here?
```suggestion
    :key="popoverKey"
```